src/Agent.Listener/Configuration/CredentialProvider.cs

```diff
@@ -42,18 +45,28 @@ public CredentialProvider(string scheme)
 
     public sealed class AadDeviceCodeAccessToken : CredentialProvider
     {
-        private string _azureDevOpsClientId = "97877f11-0fc6-4aee-b1ff-febb0519dd00";
+        private VssCredentials _cacheCreds = null;
 
-        public override Boolean RequireInteractive => true;
+        //Default Application (Client) Id
+        private readonly string _clientId = "97877f11-0fc6-4aee-b1ff-febb0519dd00";
 
+        //Default ADO resource ID for user impersonation scope
```

Comment:
nit: this is not a default - as there's no way to override these, consider just dropping these two confusing comments.

Reply:
Removed